app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -408,6 +470,34 @@ def self.host_inv_to_lan_hashes(inv, switch_uids)
           result_uids[uid] = new_result
           switch[:lans] << new_result
         end
+
+        dvportgroup_inv.to_miq_a.each do |data|
+          spec = data['config']
+          next if spec.nil?
+
+          # Find the switch to which this lan is connected
+          switch = switch_uids[spec['distributedVirtualSwitch']]
```

Comment:
That is returning some information in `inv[:dvswitch]` but we still need to update VimPropMap to return the additional info you're requesting.  I'll push a commit that fixes that.